frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineRunLogs.tsx

```diff
@@ -51,14 +51,14 @@ class PipelineRunLogs extends React.Component<PipelineRunLogsProps, PipelineRunL
   getSortedTaskRun = (taskRunFromYaml) => {
     const taskRuns = Object.keys(taskRunFromYaml).sort((a, b) => {
       if (_.get(taskRunFromYaml, [a, 'status', 'completionTime'], false)) {
-        return taskRunFromYaml[b].status.completionTime &&
+        return taskRunFromYaml[b].status?.completionTime &&
           new Date(taskRunFromYaml[a].status.completionTime) >
             new Date(taskRunFromYaml[b].status.completionTime)
           ? 1
           : -1;
       }
       return taskRunFromYaml[b].status.completionTime ||
-        new Date(taskRunFromYaml[a].status.startTime) >
+        new Date(taskRunFromYaml[a].status?.startTime) >
           new Date(taskRunFromYaml[b].status.startTime)
         ? 1
```

Comment:
You didn't quite implement this the same way, but I see `.startTime` is being optional in 4.5 & 4.6

Can we not just mirror what we have there so there are no surprises in this backport. Optionality should be pretty safe here.

![image](https://user-images.githubusercontent.com/8126518/92247574-6a595a80-ee95-11ea-9ec8-c3ecbc965e43.png)
From https://github.com/openshift/console/pull/6053

Reply:
updated

Reply:
So the change here is now bigger then the origin PR? I think this one is good, but should we change the master also and replace `status.startTime` with `status?.startTime` ?? :thinking:

Reply:
@jerolimov The master works fine and tested well along the scenarios. This is just an overprotection so that things do not break.

Reply:
changed it to be same as master

Reply:
I think this is fine... I'll let @jerolimov comment as he had a pending question that he has not responded to the answer.

Reply:
Its now similar to master so this cherrypick is for me. :+1: